src/services/sync/strategies/CSVSyncer.js

```diff
@@ -394,6 +399,30 @@ class CSVSyncer extends mix(Syncer).with(ClassImporter) {
 	static splitClasses(classes) {
 		return classes.split('+').filter((name) => name !== '');
 	}
+
+	static dateValidator(dateInput) {
```

Comment:
please consider a different naming and interfce for this function, something like `isValidDate(smth) => boolean` or `assertDateFormat(smth) => throws exception`, etc.

Don't use strings as return type!